easybuild/tools/rpath.py

```diff
@@ -0,0 +1,57 @@
+"""A module to setup and use an ld wrapper to support RPATH"""
+
+import os
+import copy
+import shutil
+import stat
+import tempfile
+from vsc.utils import fancylogger
+from easybuild.tools.filetools import adjust_permissions
+
+_log = fancylogger.getLogger('tools.package')
+
+orig_os_environ = copy.deepcopy(os.environ)
```

Comment:
we have copies of the original environment in various places already, no need to keep another?